commands/quote.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+const db = require('../services/db');
+
+module.exports = {
+  db_required: true,
+  message_regex: /^.(?:quote)(?: ([^ ]+)|$)(?: (.+)|$)$/,
+  allow: ({ isMod, isAuthenticated, isPM }) => isMod && isAuthenticated && !isPM,
+  response({ message_match: [, command, quote], author_match: [author], channel }) {
+    if (!command) {
+      command = 'get';
+    }
+    switch (command.toLowerCase()) {
+      case 'add':
+      case 'store':
+        return add(author, channel, quote);
+      case 'get':
+        return get(channel, quote);
+      case 'search':
+        return search(channel, quote);
+      case 'help':
+        return [
+          'Add \'quote\' to add a new quote',
```

Comment:
Can we make this `.quote add \'quote\' to add a new quote` to make the usage clearer? Same with the other help output.

Reply:
Done